web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -122,19 +121,19 @@ export namespace TaskItem {
         case "sms":
           return html`
             <md-badge color="darkmint" circle>
-              <md-icon name="social-sms" size="20" iconSet="momentumDesign" color="white-100"></md-icon>
+              <md-icon name="sms-filled" size="20" iconSet="momentumDesign" color="white-100"></md-icon>
             </md-badge>
           `;
         case "facebook":
           return html`
             <md-badge bgColor="#0078FF" circle>
-              <md-icon name="social-fbmessenger" size="20" iconSet="momentumDesign" color="white-100"></md-icon>
+              <md-icon name="messenger_16" iconSet="momentumUI" color="white-100"></md-icon>
             </md-badge>
```

Comment:
hardcode to momentumUI instead of brand visual as the -bw variant is hardcoded to black